### ISSUE TITLE
Features that would be nice to have (Drafts (posts/comments) | Direct image upload | Direct reply (me tab) | Local multis | Searching sort

### ISSUE THREAD

**Issue by iiTzRichie** [lo-cafe/winston#128]
Some features that would be nice to have:

• Since drafts are kind of an automatic thing and your post/comment is automatically saved if you swipe out, it would be nice to have a little indicator that lets you know that you have drafts saved.

• A direct Reply option within the Inbox would be nice to have.

• Direct image uploading would be nice to have.

• Currently your Multis created on Reddit are imported to Winston, but it would be nice to be able to create Multis within Winston itself.

• Sorting options when searching in a subreddit and/or searching Reddit as a whole.

**Comment by ojsef39** [comment]
@iitzRichie thank you very much for the feature requests, but for better clarity I would be grateful if you would not open megathreads next time since that makes it harder to find a specific feature request again. Thanks ❤️

**Comment by iitzRichie** [comment]
> Thank you very much for the feature requests, but for better clarity I would be grateful if you would not open megathreads next time since that makes it harder to find a specific feature request again. Thanks ❤️ 

True. My bad. I just like to compile a list rather than doing one at a time.

**Comment by ojsef39** [comment]
btw @iitzRichie feel free to create megathreads in the discussion tabs under ideas. from there it would be easier to manage :)

**Comment by iitzRichie** [comment]
> btw @iitzRichie feel free to create megathreads in the discussion tabs under ideas. from there it would be easier to manage :)

Should we move Feature Requests over there then, or still keep them mixed in with Bug Reports in the Issues tab?
